Incompatibility with React Native 0.65

It seems that [changes to getEventData()](https://github.com/facebook/react-native/commit/72d0ddc16f2f631003c3486e0a59e50c145ec613) were made in 0.65 that may break compatibility with this lib. This isn't showstopping as 0.65 is only at rc0 currently, but I wanted to flag it early.

```
node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/PickerFocusEvent.java:24: error: getEventData() in com.reactnativecommunity.picker.PickerFocusEvent cannot override getEventData() in com.facebook.react.uimanager.events.Event
  private WritableMap getEventData() {
                      ^
  attempting to assign weaker access privileges; was protected
node_modules/@react-native-picker/picker/android/src/main/java/com/reactnativecommunity/picker/PickerBlurEvent.java:24: error: getEventData() in com.reactnativecommunity.picker.PickerBlurEvent cannot override getEventData() in com.facebook.react.uimanager.events.Event
  private WritableMap getEventData() {
                      ^
  attempting to assign weaker access privileges; was protected
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
2 errors
```

--- a/android/src/main/java/com/reactnativecommunity/picker/PickerBlurEvent.java
+++ b/android/src/main/java/com/reactnativecommunity/picker/PickerBlurEvent.java
@@ -21,7 +21,17 @@ public class PickerBlurEvent extends Event<PickerBlurEvent> {
     rctEventEmitter.receiveEvent(getViewTag(), getEventName(), getEventData());
   }
 
-  private WritableMap getEventData() {
+  /**
+   * In RN version 0.65+ getEventData method is introduced
+   * https://github.com/facebook/react-native/commit/72d0ddc16f2f631003c3486e0a59e50c145ec613
+   *
+   * In order to keep compatibility with 0.65+ and not introduce breaking change for that lib,
+   * the only change is changing access modifier from `private` to `protected` without using @override annotation
+   * (idk if it will not bring any "method is hidden, but should overriden" warning, but for now it just works)
+   *
+   * In future versions, where `dispatch` method will be removed, there will be need to make a breaking change here
+   */
+  protected WritableMap getEventData() {
     WritableMap eventData = Arguments.createMap();
     eventData.putInt("target", getViewTag());
     return eventData;

--- a/android/src/main/java/com/reactnativecommunity/picker/PickerFocusEvent.java
+++ b/android/src/main/java/com/reactnativecommunity/picker/PickerFocusEvent.java
@@ -21,7 +21,17 @@ public class PickerFocusEvent extends Event<PickerFocusEvent> {
     rctEventEmitter.receiveEvent(getViewTag(), getEventName(), getEventData());
   }
 
-  private WritableMap getEventData() {
+  /**
+   * In RN version 0.65+ getEventData method is introduced
+   * https://github.com/facebook/react-native/commit/72d0ddc16f2f631003c3486e0a59e50c145ec613
+   *
+   * In order to keep compatibility with 0.65+ and not introduce breaking change for that lib,
+   * the only change is changing access modifier from `private` to `protected` without using @override annotation
+   * (idk if it will not bring any "method is hidden, but should overriden" warning, but for now it just works)
+   *
+   * In future versions, where `dispatch` method will be removed, there will be need to make a breaking change here
+   */
+  protected WritableMap getEventData() {
     WritableMap eventData = Arguments.createMap();
     eventData.putInt("target", getViewTag());
     return eventData;

